scripts/reana_bench.py

```diff
@@ -358,8 +473,8 @@ def _build_submitted_results_path(workflow: str) -> Path:
     return Path(f"{workflow}_submitted_results.csv")
 
 
-def _build_processed_results_path(workflow: str) -> Path:
-    return Path(f"{workflow}_processed_results.csv")
+def _build_derived_results_path(workflow: str) -> Path:
+    return Path(f"{workflow}_analyzed_results.csv")
```

Comment:
When you analyse a subset, e.g. `analyze -n N1-N2`, then the file is still named `analyzed_results.csv`, but it contains only the subset of runs corresponding to N1-N2, not the full set. So it should rather be named `analyzed_results_N1_N2.csv`, quite like the plots?

Reply:
I calculate metrics (`runtime`, `pending`, etc.) on the whole file not only subset. I filter later and produce plots with N1_N2. So putting `N1_N2` to `analyzed_results` will be misleading at this point.

Reply:
Fully agree, sorry my mistake, I confused files :man_facepalming:

Reply:
No problems. 

Right now, it is just easier to collect everything together and filter after. We may revisit it later.